server/src/initializers/AppInitializer.ts

```diff
@@ -50,6 +52,22 @@ function configureRoutes(db: Db, express: expressServer.Express): void {
   express
     .route("/api/runewords/:runewordName")
     .get(itemsController.getRuneword);
+
+  express.route("/api/party").post(partyController.add);
+
+  express.route("/api/party/:address").get(partyController.get);
+
+  express
+    .route("/api/party/:address/signup")
+    .put(partyController.signupToParty);
+
+  express
+    .route("/api/party/:address/manage/:method")
+    .put(partyController.updatePartyUser);
+
+  express
+    .route("/api/party/:address/password/validate")
+    .put(partyController.validatePassword);
 }
```

Comment:
this file should be cleaned up and splitted in some methods (`registerGrailRoutes` etc.). However also something for later => issue

Reply:
https://github.com/Nasicus/d2-holy-grail/issues/28